packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -524,13 +527,36 @@ QUnit.module('decorators markup', {}, () => {
             selectAllText: 'Test'
         }));
 
-        const $multipleContainer = $list.find('.dx-list-select-all');
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
         assert.equal($multipleContainer.length, 1, 'container for SelectAll rendered');
         assert.equal($multipleContainer.text(), 'Test', 'select all rendered');
         const $checkbox = $multipleContainer.find('.dx-checkbox');
         assert.equal($checkbox.length, 1, 'checkbox rendered');
     });
 
+    QUnit.test('selectAll text and aria-label attribute should be equal to custom localized text (T1239880)', function(assert) {
+        localization.loadMessages({ 'en': { 'dxList-selectAll': 'custom-select-all' } });
```

Comment:
The test asserts will be a bit more intuitive of we store this value to a variable.
```javascript
const localizedSelectAllText = 'custom-select-all';
localization.loadMessages({ 'en': { 'dxList-selectAll': localizedSelectAllText } });
// ...

assert.equal($selectAllCheckBox.attr('aria-label'), localizedSelectAllText, 'selectAll checkbox aria-label should be equal to localized text');
```

Reply:
[c38c1ec](https://github.com/DevExpress/DevExtreme/pull/27664/commits/c38c1ecfd87f6a8d044e3a867871ffbfa7f3fa25)